### PR TITLE
#1827 currently selected tool with mouse cursor

### PR DESCRIPTION
**Pull request by ansivgit**
Add Atom hoverIcon when atom clicked on the right toolbar. You can see chosen atom icon when you put mouse on the canvas

### CODE REVIEW

File: packages/ketcher-react/src/script/editor/Editor.ts [hunk 1]
```diff
@@ -200,6 +206,11 @@ class Editor implements KetcherEditor {
 
     const tool = new toolMap[name](this, opts)
 
+    // hide icon if not AtomToll chosen
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/editor/Editor.ts:209]: Please, fix a typo `Toll` :)

File: packages/ketcher-react/src/script/editor/Editor.ts [hunk 2]
```diff
@@ -109,6 +110,7 @@ class Editor implements KetcherEditor {
   historyPtr: any
   errorHandler: ((message: string) => void) | null
   highlights: Highlighter
+  hoverIcon: any
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/editor/Editor.ts:113]: Can't we use more specific type?

File: packages/ketcher-react/src/script/editor/tool/atom.ts [hunk 3]
```diff
@@ -129,7 +139,11 @@ class AtomTool {
 
   mousemove(event) {
     const rnd = this.editor.render
+    const { layerX, layerY } = event
+
     if (!this.dragCtx || !this.dragCtx.item) {
+      // magic numbers to adjust the position of the icon relative to the mouse pointer
+      this.editor.hoverIcon.attr({ x: layerX - 9, y: layerY - 7 })
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/editor/tool/atom.ts:146]: Can't we measure it? :) 
May be functions like `getBBox()` or `getBoundingClientRect()`?

**Review by Nitvex — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/editor/Editor.ts (hunk 1 above)
Comment (Nitvex) [reply packages/ketcher-react/src/script/editor/Editor.ts:209]: But this kind of comments we can replace with:
```
const isAtomToolChosen = name === 'atom'
if (!isAtomToolChosen) {
  ...
}
```
I think this makes the same sense.

File: packages/ketcher-react/src/script/editor/tool/atom.ts (hunk 3 above)
Comment (KonstantinEpam23) [reply packages/ketcher-react/src/script/editor/tool/atom.ts:146]: or at least put these int constants?

**Review by Nitvex — APPROVED** [review]

**Review by KonstantinEpam23 — APPROVED** [review]

**Pull request merged**